doc/system_requirements.md

```diff
@@ -19,111 +19,160 @@ Goals of this plugin:
+```
 
 Covers:
 
-* [feat~verify-repo-setup~1](#verify-repository-structure)
+* `req~verify-changes_x.x.x.md-file~1`
 
 Needs: dsn
 
-### Checking Maven Plugins
+#### Verify dependencies.md File
 
-`req~verify-repo-structure~1`
+`req~verify-dependencies-file~1`
+
+PK verifies that the repository contains a `dependencies.md` file that contains a Markdown table with all dependencies (name + link), and their licenses (name + link). PK groups the table into sections for compile, runtime, test anbd plugin dependencies.
 
-This plugin checks if the project has configured a minimum set of maven plugins.
+The dependencies table also includes the build plugins.
 
 Covers:
 
-* [feat~verify-repo-setup~1](#verify-repository-structure)
+* [feat~verify-repo-setup~1](#verify-repository)
 
 Needs: dsn
 
-### Fixing Repository Structure
+#### Verify That Files Exist
 
-`req~fix-repo-structure~1`
+`req~verify-that-files-exist~1`
 
-This plugin provides a command that automatically creates or updates the files so that they fit the required project structure.
+PK verifies that certain files exist in the repository (with an arbitrary content).
+
+Examples: `README.md`, `travis.yml`.
 
 Covers:
 
-* [feat~fix-repo-setup~1](#fix-repository-structure)
+* [feat~verify-repo-setup~1](#verify-repository)
 
 Needs: dsn
 
-### Verify changes file
+#### Verify That Files do not Exist
 
-`req~verify-changes-file~1`
+`req~verify-that-files-do-not-exist~1`
 
-PK makes sure that the project contains a `doc/changes/changes_X.X.X.md` file.
+PK verifies that certain files do not exist in the repository.
 
-Needs: impl, itest, utest
+Example: Broken-Link checker GitHub workflow
+
+Rationale:
+
+From time to time we rename a required file. In that case, PK should make sure that the old version of the file no longer exists in the repositories.
 
 Covers:
 
-* [feat~verify-repo-setup~1](#verify-repository-structure)
+* [feat~verify-repo-setup~1](#verify-repository)
 
-### Verify dependencies file
+Needs: dsn
 
-`req~verify-dependencies-file~1`
+### Fix Findings
 
-PK makes sure that the repository contains a `dependencies.md`  that contains a Markdown table with all dependencies (name + link) and their licenses (name + link).
+`feat~fix-findings~1`
 
-Covers:
+Wherever possible, PK can automatically fix the validation findings.
+
+Rationale:
+
+Fixing the findings by hand would be a wast of time.
+
+Needs: dsn
+
+### Configuration
+
+`feat~configuration~1`
+
+Users can configure the project template to a certain degree.
+
+Rationale:
+
+Different projects require different project structures. For example projects that are released on maven central need more maven plugin than other projects.
+
+Needs: dsn
+
+### Maven Integration
+
+`feat~mvn-integration~1`
 
-* [feat~verify-repo-setup~1](#verify-repository-structure)
+PK can be integrated in the maven lifecycle. By that it can break the CI build if the project structure is invalid.
```

Comment:
```suggestion
PK can be integrated into the maven lifecycle. By that, it can break the Continuous Integration (CI) build if the project structure is invalid.
```